README.md

```diff
@@ -182,3 +209,7 @@ maintainable solution.
 
 Please double-check the [INSTALL](INSTALL.md) documents, and search issue archives. If 
 this doesn't help, please post your question using GitHub Issues.
+
+### Is there a simpler and more productive way for writing wxRuby3 desktop GUI applications?
```

Comment:
Could you make this read:  "Is there another, more declarative way, for writing wxRuby3 desktop GUI applications?

Reply:
Done.